.github/workflows/cache-speed-test.yml

```diff
@@ -47,7 +48,7 @@ jobs:
           path: |
             modules/
             zephyr/
-          key: zephyr-modules-${{ env.cache-name }}-${{ hashFiles('zephyr/west.yml') }}
+          key: zephyr-modules-${{ env.cache-name }}-${{ hashFiles('~/zephyrproject/zephyr/west.yml') }}
```

Comment:
You could also do something like `~/**/zephyr/west.yml` or not use `~` at all in the `west init` step and then your initial syntax of `**/zephyrproject/zephyr/west.yml` should be fine.